src/huggingface_hub/keras_mixin.py

```diff
@@ -144,6 +151,172 @@ def _create_model_card(
         f.write(readme)
 
 
+class PushToHubCallback(Callback):
+    """
+    Callback that will save and push Keras models to the Hub regularly. By default, it pushes once per epoch, but this can
```

Comment:
```suggestion
    Callback that will periodically save and push Keras models to the Hugging Face Hub. By default, it pushes once per epoch, but this can
```